src/opcodes/tests/test_stack.cairo

```diff
@@ -291,6 +291,17 @@ fn test_op_2rot_insufficient_items() {
 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_1 OP_2ROT";
```

Comment:
Could you change this to use a for loop instead to create the string. You should be able to do something like 
```
let mut program = "";
let op_1_string = "OP_1 ";
// loop
program.append(op_1_string)
```

Reply:
can we apppend a value to ByteArray

Reply:
we cannot append a value to string, as append method only works if it an array